content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -79,4 +79,33 @@ Refer to the [CSI specifications](https://github.com/container-storage-interface
 
 * For volumes managed by in-tree plugins that have been migrated to a CSI driver, the maximum number of volumes will be the one reported by the CSI driver.
 
+### Mutable CSI Node Allocatable Count
+
+{{< feature-state state="alpha" for_k8s_version="v1.33" >}}
+
+CSI drivers can dynamically adjust the maximum number of volumes that can be attached to a Node at runtime. This enhances scheduling accuracy and reduces pod scheduling failures due to changes in resource availability.
+
+This is an alpha feature and is disabled by default.
+
+To use this feature, you must enable the `MutableCSINodeAllocatableCount` feature gate on the following components:
+
+- `kube-apiserver`
+- `kubelet`
+
```

Comment:
Doesn't this also need to be enabled in scheduler too?

Reply:
hmm, actually I just realized the code in scheduler is not behind a feature-gate.